models/public/efficientnet-b0-pt/efficientnet-b0-pt.md

```diff
@@ -0,0 +1,78 @@
+# efficientnet-b0-pt
+
+## Use Case and High-Level Description
+
+The `efficientnet-b0-pt` model is one of the [efficientnet](https://arxiv.org/abs/1905.11946) group of models designed to perform image classification. This model was pretrained in PyTorch\* All the EfficientNet models have been pretrained on the ImageNet image database. For details about this family of models, check out the [repository](https://github.com/rwightman/gen-efficientnet-pytorch).
```

Comment:
For the article link text, I think you should use the proper capitalization (EfficientNet). Also, there's a period missing after "PyTorch".